packages/modules/storybook-auth-addon/src/index.js

```diff
@@ -1,3 +1,3 @@
-import { BrowserAuthorizationCallbackHandler } from "@bentley/frontend-authorization-client";
+import { BrowserAuthorizationClient } from "@itwin/browser-authorization";
 
-BrowserAuthorizationCallbackHandler.handleSigninCallback(window.location.toString()).catch(console.log);
+BrowserAuthorizationClient.handleSignInCallback().catch(console.log);
```

Comment:
Was talking with @raplemie and he was wondering if we need the addon or if we can get away without it and put the auth directly in the app. @ben-polinsky thoughts?

Reply:
Have to get back to this. Yeah I could look into that when I have a chance.